pkgs/by-name/mg/mgitstatus/package.nix

```
@@ -0,0 +1,33 @@
+{ fetchFromGitHub, lib, stdenv }:
+
+stdenv.mkDerivation rec {
+  pname = "mgitstatus";
+  version = "2.2";
+
+  src = fetchFromGitHub {
+    owner = "fboender";
+    repo = "multi-git-status";
+    rev = version;
+    hash = "sha256-jzoX7Efq9+1UdXQdhLRqBlhU3cBrk5AZblg9AYetItg=";
+  };
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/bin $out/man/man1
+    install -m 755 mgitstatus $out/bin/${pname}
+    install mgitstatus.1 $out/man/man1
```

suggestion: no need for `mkdir -p`

```suggestion
    install -Dm 755 mgitstatus -t $out/bin
    install -D mgitstatus.1 -t $out/man/man1
```

---

Consider using `installManPage mgitstatus.1`.

---

> Consider using `installManPage mgitstatus.1`.

I cannot find a command like that? Can you reference it? Even so, I'd rather not add an extra dependency for no real benefit.

---

It's in [`installShellFiles`](https://nixos.org/manual/nixpkgs/stable/#installshellfiles), which you can add to the `nativeBuildInputs`. I don't think adding that as a build dependency is the end of the world.
It installs to the correct location (which is `$out/share/man`) and sets appropriate permissions.

A small quibble: Based on the documentation, manpages are supposed to be installed from the `postInstall` hook.